### ISSUE TITLE
Enforcement Screen - VoiceOver Does Not Read Highlighted Text Correctly on the "Criminal" Tab

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#134]
While looking at the iOS version of Sweat and Toil on the "Enforcement" screens "Criminal" tab VoiceOver does not read highlighted text starting with "Imposed Penalties..." but the text "Referral Mechanism Exists..." instead then reads "Referral Mechanism Exists..." when it gets to that item on the screen.

It's as though VoiceOver is reading the wrong text for what is displayed/highlighted.

Please see the attached screenshot.

![ilab - sweat and toil - ios - VoiceOver does not read highlighted text but the text below that item and then again once on that item](https://user-images.githubusercontent.com/10423487/96150797-bc4fc000-0ed8-11eb-81a7-fb7a8aba6b67.png)

**Comment by jfc3-dol** [comment]
@pbhatt17 

I am closing the issue because VoiceOver is reading Burma's and other random "Enforcement" screens "Criminal" tab as expected.
